Solutions/Marain.TenantManagement.Abstractions/Marain/TenantManagement/ITenantManagementService.cs

```diff
@@ -93,13 +94,23 @@ public interface ITenantManagementService
             ITenant serviceTenant,
             EnrollmentConfigurationItem[] configurationItems);
 
+        /// <summary>
+        /// Unenrolls the specified tenant from the service.
+        /// </summary>
+        /// <param name="enrolledTenantId">The tenant that is currently enrolled.</param>
+        /// <param name="serviceTenantId">The service they need to be unenrolled from.</param>
+        /// <returns>A task which completes when the unenrollment has finished.</returns>
+        Task UnenrollFromServiceAsync(
```

Comment:
As a general comment in this area, I like to leave the interface as small as possible, and where possible add extension methods to the interface to implement "overloads" (e.g. one that takes and `ITenant` and one that takes a `TenantId` - often one of those can become an extension method)